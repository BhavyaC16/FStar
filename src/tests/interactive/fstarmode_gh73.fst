module FStarMode_GH73
open FStar.Tactics

let x: bool = synth_by_tactic (exact (quote 1))
