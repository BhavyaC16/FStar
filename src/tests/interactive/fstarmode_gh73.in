{"query-id":"1","query":"push","args":{"kind":"full","code":"module Test\nopen FStar.Tactics\n","line":1,"column":0}}
{"query-id":"2","query":"peek","args":{"kind":"lax","code":"\nlet x: bool = synth_by_tactic (exact (\"A\" + 1))\n","line":3,"column":0}}
{"query-id":"3","query":"push","args":{"kind":"full","code":"\nlet x: bool = synth_by_tactic (exact (\"A\" + 1))\n","line":3,"column":0}}
{"query-id":"4","query":"peek","args":{"kind":"lax","code":"\nlet x: bool = synth_by_tactic (exact (quote 1))\n","line":3,"column":0}}
{"query-id":"5","query":"push","args":{"kind":"full","code":"\nlet x: bool = synth_by_tactic (exact (quote 1))\n","line":3,"column":0}}
