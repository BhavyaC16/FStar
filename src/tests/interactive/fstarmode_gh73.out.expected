{"kind": "protocol-info", "rest": "[...]"}
{"kind": "response", "query-id": "1", "response": [{"level": "warning", "message": "Pattern uses these theory symbols or terms that should not be in an smt pattern:  Prims.eq2 Prims.eq2", "ranges": [{"beg": [1010, 2], "end": [1010, 136], "fname": "FStar.List.Tot.Properties.fst"}]}, {"level": "warning", "message": "Pattern uses these theory symbols or terms that should not be in an smt pattern:  Prims.op_Addition 1", "ranges": [{"beg": [939, 2], "end": [939, 49], "fname": "FStar.Seq.Properties.fst"}]}, {"level": "warning", "message": "Pattern uses these theory symbols or terms that should not be in an smt pattern:  0", "ranges": [{"beg": [958, 2], "end": [958, 33], "fname": "FStar.Seq.Properties.fst"}]}, {"level": "warning", "message": "Pattern uses these theory symbols or terms that should not be in an smt pattern:  Prims.eq2 Prims.op_Modulus 0", "ranges": [{"beg": [625, 8], "end": [625, 66], "fname": "FStar.UInt.fst"}]}, {"level": "warning", "message": "Pattern uses these theory symbols or terms that should not be in an smt pattern:  Prims.eq2 Prims.op_Modulus 0", "ranges": [{"beg": [626, 8], "end": [626, 23], "fname": "FStar.UInt.fst"}, {"beg": [625, 8], "end": [625, 66], "fname": "FStar.UInt.fst"}]}, {"level": "warning", "message": "Pattern uses these theory symbols or terms that should not be in an smt pattern:  Prims.op_LessThan", "ranges": [{"beg": [644, 8], "end": [644, 61], "fname": "FStar.UInt.fst"}]}, {"level": "warning", "message": "Pattern uses these theory symbols or terms that should not be in an smt pattern:  Prims.op_LessThan", "ranges": [{"beg": [645, 8], "end": [645, 22], "fname": "FStar.UInt.fst"}, {"beg": [644, 8], "end": [644, 61], "fname": "FStar.UInt.fst"}]}, {"level": "warning", "message": "Pattern uses these theory symbols or terms that should not be in an smt pattern:  Prims.op_Subtraction 1", "ranges": [{"beg": [664, 8], "end": [664, 51], "fname": "FStar.UInt.fst"}]}, {"level": "warning", "message": "Pattern uses these theory symbols or terms that should not be in an smt pattern:  Prims.op_Subtraction 1", "ranges": [{"beg": [665, 8], "end": [665, 25], "fname": "FStar.UInt.fst"}, {"beg": [664, 8], "end": [664, 51], "fname": "FStar.UInt.fst"}]}], "status": "success"}
{"kind": "response", "query-id": "2", "response": [{"level": "error", "message": "Expected expression of type \"int\"; got expression \"\"A\"\" of type \"string\"", "ranges": [{"beg": [4, 50], "end": [4, 51], "fname": "<input>"}]}], "status": "success"}
{"kind": "response", "query-id": "3", "response": [{"level": "error", "message": "Expected expression of type \"int\"; got expression \"\"A\"\" of type \"string\"", "ranges": [{"beg": [4, 50], "end": [4, 51], "fname": "<input>"}]}], "status": "failure"}
{"kind": "response", "query-id": "4", "response": [], "status": "success"}
{"contents": {"depth": 1, "goals": [{"goal": {"label": "", "type": "bool", "witness": "(*?u[...]*) _"}, "hyps": []}], "label": "at the time of failure", "location": "<location removed>", "smt-goals": []}, "kind": "message", "level": "proof-state", "query-id": "5"}
{"kind": "response", "query-id": "5", "response": [{"level": "error", "message": "user tactic failed: exact: 1 : int does not exactly solve the goal bool (witness = (*?u[...]*) _)", "ranges": [{"beg": [4, 14], "end": [4, 57], "fname": "<input>"}, {"beg": [59, 12], "end": [59, 16], "fname": "FStar.Tactics.Derived.fst"}]}], "status": "failure"}
